dials.go

```diff
@@ -156,30 +156,11 @@ type sourceValue struct {
 	value  reflect.Value
 }
 
+// Type is a wrapper with for a reflect.Type.
```

Comment:
I think you have an extra `with` in this comment.

Reply:
Indeed.  :-(